core-processor/src/ext.rs

```diff
@@ -207,6 +210,15 @@ impl EnvExt for Ext {
         self.message_context.current().source()
     }
 
+    fn exit(&mut self, address: ProgramId) -> Result<(), &'static str> {
```

Comment:
Isn't better to keep common names across the code?
```suggestion
    fn exit(&mut self, value_destination: ProgramId) -> Result<(), &'static str> {
```